_docs/usage.md

```diff
@@ -93,6 +93,18 @@ run:
 THELOUNGE_HOME=/tmp thelounge start
 ```
 
+## Installing additional themes
+
+A list of all available themes can be found [on the npm registry](https://www.npmjs.com/search?q=keywords%3Athelounge-theme). To install a theme called `thelounge-theme-foo`, run:
```

Comment:
Change `foo` to `solarized`, so that the example is real (and points to a package we control)

Reply:
Done in 9f9fd79